apps/cli/src/commands/auth.command.ts

```diff
@@ -67,6 +67,7 @@ export class AuthCommand extends Command {
 				'Authentication token (optional, for SSH/remote environments)'
 			)
 			.option('-y, --yes', 'Skip interactive prompts')
+			.option('--no-header', 'Suppress the Task Master header banner')
```

Comment:
the reason why I made this an env variable and not a flag is because I don't think these flags should be user-facing, user doesn't need to know about this flag when doing a --help